src/redux/actions/experiments/createExperiment.js

```diff
@@ -15,8 +15,8 @@ import handleError from 'utils/http/handleError';
 const createExperiment = (
   name, description,
 ) => async (dispatch) => {
-  const createdDate = moment().toISOString();
-  const experimentId = hash.MD5(createdDate);
+  const createdAt = moment().toISOString();
+  const experimentId = hash.MD5(createdAt);
```

Comment:
Weren't we going to use a UUIDV4 to generate the experiment ID instead of using MD5 hash?

Reply:
yes, but that's part of a differnet ticket because it involves s3 migrations